news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "Acknowledging Community Contributors"
+date: 2019-02-22 00:00:00 +0000
+---
+
+At its very heart, osu! is almost entirely driven by its community. And at the heart of our community, the efforts of an outstanding few make themselves known. Today, we take a moment to reflect on some of these people, and award them the recognition they deserve.
+
+![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/banner.jpg)
+
+Perhaps the most prestigious of all profile badges awarded, the [community contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg) is expressly granted to those of the osu! community who have gone utterly above and beyond the norm in their enduring contribution to the well-being of the game and its community.
+
+In a first for many years, we are proud to announce the latest set of community contributors in what is the largest number of additions made since the initiative started back in 2014.
+
+Please note however - this list is by no means exhaustive, and only represents the acknowledgement of continued, consistent contribution across a long period of time as observed and noted by the osu!team.
+
+## Community
+
+### HappyStick
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/HappyStick.jpg)](https://osu.ppy.sh/u/256802)
```

Comment:
use `/users` links (unless if there was a specific reason to use short ones here?)

Reply:
`/u` is the old site.